docs/migration.adoc

```diff
@@ -0,0 +1,60 @@
+== Migration
+
+We have chosen to migrate away from using Alpine based Docker images and to using Ubuntu. The reason for this is that we need to be able to support multi-platform images:
+
+* Intel based machines (AMD64)
+* Apple Silicon machines (ARM64)
+
+Although there are ARM versions of Alpine and PowerShell, Microsoft have not released PowerShell for Alpine ARM64. This has meant that it is not possible to run the EIR Docker images on an Apple Silicon based Mac. We have now moved away from Alpine and use Ubuntu which can be easily built for the two targeted platforms.
```

Comment:
Minor thing, it's possible to but it's not supported by MS currently so any issues we have aren't supportable...

Reply:
Updated